src/measure_theory/l1_space.lean

```diff
@@ -797,6 +814,33 @@ variables {𝕜 : Type*} [normed_field 𝕜] [normed_space 𝕜 β]
 lemma smul_to_fun (c : 𝕜) (f : α →₁[μ] β) : ⇑(c • f) =ᵐ[μ] c • f :=
 ae_eq_fun.coe_fn_smul _ _
 
+lemma norm_eq_lintegral {f : α →₁[μ] β} : ∥f∥ = (∫⁻ x, (nnnorm (f x) : ennreal) ∂μ).to_real :=
+by simp [l1.norm_eq, ae_eq_fun.edist_zero_eq_coe, ← edist_eq_coe_nnnorm]
+
+lemma norm_sub_eq_lintegral {f g : α →₁[μ] β} :
+  ∥f - g∥ = (∫⁻ x, (nnnorm (f x - g x) : ennreal) ∂μ).to_real :=
```

Comment:
Is it defeq `norm_eq_lintegral (f - g)`? BTW, why use implicit args?

Reply:
No, it's not, since `f - g` and `λ x, f x - g x` are not equal, but only a.e. equal. I will make the arguments explicit,